util.js

```diff
@@ -2,43 +2,32 @@
 
 exports.stringToBytes = string => [...string].map(character => character.charCodeAt(0));
 
-exports.tarHeaderChecksumMatches = buffer => { // Does not check if checksum field characters are valid
-	if (buffer.length < 512) { // `tar` header size, cannot compute checksum without it
-		return false;
-	}
+/**
+ * Return true if the tar checksum is valid
+ * @param {Buffer} buffer holding tar header [offset ... offset + 512]
+ * @param {number} off TAR header offset
+ * @returns {boolean} rue if the tar checksum is valid, otherwise false
+ */
+exports.tarHeaderChecksumMatches = (buffer, off) => {
```

Comment:
```suggestion
exports.tarHeaderChecksumMatches = (buffer, offset = 0) => {
```

Reply:
I thought this was only possible in Typescript, good to know!.